workers/miscale.py

```diff
@@ -41,12 +34,9 @@ def status_update(self):
         if results.midatetime:
             messages.append(MqttMessage(topic=self.format_topic("midatetime"), payload=results.midatetime))
         
-        if self.is_users:
+        if hasattr(self, 'users'):
             for key, item in self.users.items():
-                greater = float(item['weight_template'].split(' to ')[0])
-                less = float(item['weight_template'].split(' to ')[1])
-                
-                if results.weight > greater and results.weight < less:
+                if results.weight > item['weight_template']['min'] and results.weight < item['weight_template']['max']:
```

Comment:
Maybe ">=" and "<=". I think more people will expect to include range ends, as least suprise